src/components/RegisterForm/index.tsx

```diff
@@ -33,17 +33,48 @@ export default function RegisterForm() {
 
 				<FormControl isRequired>
 					<FormLabel>First name</FormLabel>
-					<Input placeholder="First name" {...register(RegisterFields.FIRST_NAME)} />
+					<Input
+						pattern="[A-Za-z]{1,32}"
+						title="First name should contain only letters."
+						placeholder="First name"
+						{...register(RegisterFields.FIRST_NAME)}
+					/>
```

Comment:
Wow, this really works? 😄 

No need to change, but this is how you can implement this with [`react-hook-form`](https://react-hook-form.com/api/useform/register).
```suggestion
					<Input
						placeholder="First name"
						{...register(RegisterFields.FIRST_NAME, {
						  required: "First name is required",
						  pattern: {
						    value: /^[a-z]*$/i,
						    message: "First name can contain only letters",
						  },
						})}
					/>
```

Underneath the input, you can use the [`ErrorMessage`](https://react-hook-form.com/api/useformstate/errormessage) component

Reply:
I added some need screenshots of how message for validation are shown.
I found this pattern thing and I liked it, but let me know if I should use ErrorMessage component instead. Maybe it's easier for user to see validation message right after they fill out each Input and not after they click on Sign up button.